src/Contrib/Service/Masa.Contrib.Service.MinimalAPIs/Infrastructure/ServiceBaseHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Service.MinimalAPIs;
+
+public static class ServiceBaseHelper
+{
+    public static Delegate CreateDelegate(MethodInfo methodInfo, object targetInstance)
+    {
+        var type = Expression.GetDelegateType(methodInfo.GetParameters().Select(parameterInfo => parameterInfo.ParameterType)
+            .Concat(new List<Type>
+                { methodInfo.ReturnType }).ToArray());
+        return Delegate.CreateDelegate(type, targetInstance, methodInfo);
+    }
+
+    public static string CombineUris(params string[] uris) => string.Join("/", uris.Select(u => u.Trim('/')));
+
+    public static string TrimMethodName(string methodName)
+        => methodName.TrimEnd("Async", StringComparison.OrdinalIgnoreCase);
+
+    public static bool TryParseHttpMethod(string[] prefixs, ref string methodName)
+    {
+        var newMethodName = methodName;
```

Comment:
Temporary variables are required, otherwise it will be considered an error by the compiler

`Cannot use 'ref' parameter 'methodName' inside lambda expression`

Reply:
Processed